libhb/qsv_common.c

```diff
@@ -892,6 +912,7 @@ hb_list_t* hb_qsv_load_plugins(hb_qsv_info_t *info, mfxSession session, mfxVersi
 
     if (HB_CHECK_MFX_VERSION(version, 1, 8))
     {
+#if !defined(SYS_LINUX)
```

Comment:
Reviewing my own PR :stuck_out_tongue: 
@sr55 and @maximd33 I had to disable this block of code on linux because MFXVideoUSER_Load failed and it seems to work without it. i.e. it finds the h.265 encoder and succeeds at encoding. Do you know why there is a difference between windows and linux here and whether there would be a better way of handling this?

Reply:
One alternative option would be to make a failure during hb_qsv_load_plugins non-fatal.  Would that work properly on Windows.  I.e. if the error value of hb_qsv_load_plugins were ignored and query_capabilities continued to attempt to call MFXVideoENCODE_Query, would it behave correctly on Windows?

Reply:
latest Linux/Windows drivers does not require MFXVideoUSER_Load()

making it non-fatal should be fine and should not break backward compatibility
for the legacy driver users, 
over time can be removed